package.json

```diff
@@ -15,37 +15,49 @@
   "scripts": {
     "start": "gulp watch",
     "build": "gulp",
-    "prepack": "npm run build"
+    "prepack": "npm run build",
+    "stylelint": "stylelint 'assets/scss/**/*.scss'",
```

Comment:
unfortunately this does not work in windows... failing with error `Error: No files matching the pattern "'assets/scss/**/*.scss'" were found.`. maybe doing `"stylelint assets/**/*.scss"` would work for everyone?

Reply:
If I don't add the ticks, it doesn't match the files located in 'assets/scss', only subfolders. Stylelint has double quotes in their doc, maybe that will work? Unfortunately I have no idea of Windows CLI limitations 😔

Reply:
I've pushed double quotes, if you have the opportunity to test it.

Reply:
works!